### ISSUE TITLE
Fix welcome screen on mobile

### ISSUE THREAD

**Issue by adriaanvanrossum** [simpleanalytics/roadmap#594]
- [ ] Keep price and plan on one line
- [ ] Remove credit card text after first screen (show only once to user)
- [ ] Fix input field styling and make more like the white buttons or the other way around

**Comment by benaltair** [comment]
Duplicate of #593

**Comment by adriaanvanrossum** [comment]
Thanks
